packages/model-viewer-effects/src/effect-composer.ts

```diff
@@ -0,0 +1,373 @@
+export const $removeClearPass = Symbol('removeClearPass');
+export const $addClearPass = Symbol('addClearPass');
+export const $effectPasses = Symbol('effectsPass');
+export const $requires = Symbol('requires');
+export const $effects = Symbol('effects');
+export const $selection = Symbol('selection');
+export const $onSceneLoad = Symbol('onSceneLoad');
+export const $resetEffectPasses = Symbol('resetEffectPasses');
+export const $userEffectCount = Symbol('userEffectCount');
+const $updateProperties = Symbol('updateProperties');
+
+/**
+ * Light wrapper around {@link EffectComposer} for storing the `scene` and `camera
+ * at a top level, and setting them for every {@link Pass} added.
+ */
+export class EffectComposer extends PPEffectComposer {
+  public camera?: Camera;
+  public scene?: ModelScene;
+  public dirtyRender?: boolean;
+
+  constructor(
+    renderer?: WebGLRenderer,
+    options?: {
+      depthBuffer?: boolean;
+      stencilBuffer?: boolean;
+      alpha?: boolean;
+      multisampling?: number;
+      frameBufferType?: number;
+    }
+  ) {
+    super(renderer, options);
+  }
+
+  /**
+   * Adds a pass, optionally at a specific index.
+   * Additionally sets `scene` and `camera`.
+   * @param pass A new pass.
+   * @param index An index at which the pass should be inserted.
+   */
+  override addPass(pass: Pass, index?: number): void {
+    super.addPass(pass, index);
+    this.refresh();
+  }
+
+  override setMainCamera(camera: Camera): void {
+    this.camera = camera;
+    super.setMainCamera(camera);
+  }
+
+  override setMainScene(scene: ModelScene): void {
+    this.scene = scene;
+    super.setMainScene(scene);
+  }
+
+  /**
+   * Effect Materials that use the camera need to be manually updated whenever the camera settings update.
+   */
+  refresh(): void {
+    if (this.camera && this.scene) {
+      super.setMainCamera(this.camera);
+      super.setMainScene(this.scene);
+    }
+  }
+
+  beforeRender(_time: DOMHighResTimeStamp, _delta: DOMHighResTimeStamp): void {
+    if (this.dirtyRender) {
+      this.scene?.queueRender();
+    }
+  }
+}
+
+export type MVPass = Pass & IntegrationOptions;
+
+export type RenderMode = 'performance' | 'quality';
+
+export class MVEffectComposer extends ReactiveElement {
+  static get is() {
+    return 'effect-composer';
+  }
+
+  /**
+   * `quality` | `performance`. Changing this after the element was constructed has no effect.
+   *
+   * Using `quality` improves banding on certain effects, at a memory cost. Use in HDR scenarios.
+   *
+   * `performance` should be sufficient for most use-cases.
+   * @default 'performance'
+   */
+  @property({ type: String, attribute: 'render-mode' })
+  renderMode: RenderMode = 'performance';
+
+  /**
+   * Anti-Aliasing using the MSAA algorithm. Doesn't work well with depth-based effects.
+   *
+   * Recommended to use with a factor of 2.
+   * @default 0
+   */
+  @property({ type: Number, attribute: 'msaa' })
+  msaa: number = 0;
```

Comment:
The above comment answers this as well

Reply:
Hmm, but you pass our `WebGLRenderer` into the postprocessing function, and it's already been set to `antialias: true`, so it seems like it must have an effect? Because once the WebGL context has been created, that can't be changed.

Reply:
@vanruesc the developer of the postprocessing library probably will be able to answer this much better than me.